core/src/main/scala/org/apache/spark/deploy/master/Master.scala

```diff
@@ -863,7 +872,29 @@ private[deploy] class Master(
     true
   }
 
-  private def decommissionWorker(worker: WorkerInfo): Unit = {
+  private def decommissionHostPorts(hostPorts: Seq[String]): Integer = {
+    val hostPortsParsed = hostPorts.map(Utils.parseHostPort)
+    val (hostPortsWithoutPorts, hostPortsWithPorts) = hostPortsParsed.partition(_._2 == 0)
```

Comment:
nit: val (hostPortsWithoutPorts, hostPortsWithPorts) = hostPorts.map(Utils.parseHostPort).partition(_._2 == 0)

Reply:
I would like to remain it as it is since I find two line statements confusing to read. Its also just 90 columns and within the 100 limit. There is also precedence for one line partition statements elsewhere.